### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by tmettes** [jbhunter52/BTWBResults#3]
It's not possible to get new data from BTWB. Is it possible to fix this?

Thanks in advance.

**Comment by jbhunter52** [comment]
Yes you are correct.  They dropped the feature to download csv files of your workout history.  So this project is no longer working.  I do not have a work around at this time.  If you find something please post back.
